feature/feature.go

```diff
@@ -33,6 +33,11 @@ const (
 	//
 	// alpha: v1.4
 	NodeAntiAffinity featuregate.Feature = "NodeAntiAffinity"
+
+	// NamespaceScopedZone is a feature gate for the NamespaceScopedZone functionality for supervisor.
+	//
+	// alpha: v1.11
+	NamespaceScopedZone featuregate.Feature = "NamespaceScopedZone"
```

Comment:
```suggestion
	NamespaceScopedZones featuregate.Feature = "NamespaceScopedZones"
```
Maybe? singular sounds a bit strange to me

Reply:
Sounds good to me. It looks more clear that the feature applies to the handling of multiple zones.